src/java.base/share/classes/sun/util/calendar/Gregorian.java

```diff
@@ -34,14 +34,14 @@
  * @since 1.5
  */
 
-public class Gregorian extends BaseCalendar {
+public final class Gregorian extends BaseCalendar {
 
-    static class Date extends BaseCalendar.Date {
-        protected Date() {
+    static final class Date extends BaseCalendar.Date {
+        Date() {
```

Comment:
I think removing `protected` from the `final` class methods is fine. Adding `@Override` may be helpful.